ui/client/components/notifications/Notification.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Dotdotdot from "react-dotdotdot";
+
+export default class Notification extends React.Component {
```

Comment:
It can be a funcitional component:) There you have some art about it https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108.

Reply:
Done